src/main/java/com/github/fppt/jedismock/datastructures/ZSetEntry.java

```diff
@@ -19,7 +19,7 @@ public ZSetEntry(double score, Slice value) {
         this.value = value;
     }
 
-    public double getScore() {
+    public Double getScore() {

```

Comment:
As `score` is `double` and is never `null`, it's best not to box it into `Double`. See my comment below

Reply:
Sorry, I was in a hurry to post the solution